web/cobrands/fixmystreet/staff.js

```diff
@@ -335,6 +335,17 @@ fixmystreet.staff_set_up = {
     });
   },
 
+  open311_category_edit: function() {
+    document.getElementById('open311_protect').addEventListener('change', function() {
```

Comment:
Yay for slowing reducing the jQuery dependency :) Sadly this will crash if `#open311_protect` doesn't exist on the page though.